public/fonts/fonts-desktop.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: 'Merriweather';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Merriweather'), url(http://localhost:41050/calypso/fonts/Merriweather.ttf) format('truetype');
```

Comment:
I was thinking about WebPack supplying proper URL according to what is currently built.
But if webpack will be bundling it, then no problem.